home/tests.py

```diff
@@ -205,3 +208,149 @@ def tags_response(self, client):
+            # User entered all the fields except Sub_Subject
+            (11, 2, "Question in History", 'How many wars Israel had?',
+             datetime(2022, 4, 7, 12, 59, 55, 4, tzinfo=pytz.UTC), 1, 4, Grade.GRADE8, None, 11, False),
+            # User entered all the fields except Book
+            (10, 1, "Question in Math", 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), 1, 4, Grade.GRADE7, 1, None, False),
+            # User entered all the fields except Book page
+        ])
+        # checks if the question was successfully inserted
+        @pytest.mark.django_db
+        def test_add_valid_question(self, valid_Input):
+            question = Question(*valid_Input)
+            question.save()
+
+            assert Question.objects.filter(pk=question.id).exists()
+
+        @pytest.mark.parametrize("invalid_Input, exception", [
+            ((10, 3, None, 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), 1, 4, Grade.GRADE7, 1, 10, False), ValidationError),
+            # User didn't enter title
+            ((10, 1, "Question in Math", None,
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), 1, 4, Grade.GRADE7, 1, 10, False), ValidationError),
+            # User did'nt enter content
+            ((10, 1, "Question in Math", 'How much is it 1+1?', "", 1, 4, Grade.GRADE7, 1, 10, False), ValidationError),
+            # The question does not contain date
+            ((10, 1, "Question in Math", 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), "", 4, Grade.GRADE7, 1, 10, False), ValidationError),
+            # User didn't choose subject
+            ((10, 1, "Question in Math", 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), 1, 4, "", 1, 10, False), ValidationError),
+            # User did'nt choose grade
+            ((10, 1, "Question in Math", 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC), 1, 4, Grade.GRADE7, 1, -10, False), ValidationError),
+            # User chose negative book number
+            ((10, 1, "Question in Math", 'How much is it 1+1?',
+             datetime(2022, 4, 7, 12, 53, 29, 4, tzinfo=pytz.UTC),
+             1, 4, Grade.GRADE7, 1, 99990, False), ValidationError),
+            # User entered to high book number
+        ])
+        # checks if the question was not inserted
+        @pytest.mark.django_db
+        def test_add_invalid_question(self, invalid_Input, exception):
+
+            with pytest.raises(exception):
+                assert Question(*invalid_Input).clean_fields()
+
+        # checks default value in is_edited
+        @pytest.mark.django_db
+        def test_default_is_edited(self, question):
+
+            assert question.is_edited is False
+
+        # checks default value in book_page once the user didn't insert a book page
```

Comment:
Instead of writing such comments, why don't you jut call the function: `test_default_book_page_is_none` ?

Comments can lie and go out of sync with the code quickly, if the code can document itself, do that instead of using comments.